docs/cluster_spec.md

```diff
@@ -585,9 +585,11 @@ You can use [external-dns](https://github.com/kubernetes-sigs/external-dns/) as
 ```yaml
 spec:
   externalDns:
-    provider: ExternalDNS
+    provider: external-dns
 ```
 
+Note that you if you have dns-controller installed, you need to remove this deployment before updating the cluster with the new configuration.
```

Comment:
can we use our pruning support to delete it instead?

Reply:
When/if it merges :)